projects/igniteui-angular/karma.conf.js

```diff
@@ -15,6 +15,7 @@ module.exports = function (config) {
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
       require('karma-jasmine-html-reporter'),
+      require('karma-junit-reporter'),
```

Comment:
Should we make a separate `karma.conf` for azure pipelines from the one for travis?

Reply:
No need. Test run will run only once. The difference here is that the result will be outputted into two different formats required for Coveralls and Azure Pipelines code coverage.
I'll play a little bit more with Coveralls and Azure Pipelines and if it's possible I'll disable Azure code coverage. In this case this junit reported won't be needed.

Reply:
Not only. Travis run is performed on `Chrome`, while the azure pipelines one on `ChromeHeadless`. Doesn't this change the setup to `ChromeHeadless` for travis as well?

Reply:
Yes, it is.